packages/ui/src/components/calendar/DayCalendar.svelte

```diff
@@ -39,44 +40,67 @@
   const dispatch = createEventDispatcher()
 
   const todayDate = new Date()
-  const cellBorder = 1
+  const ampm = new Intl.DateTimeFormat([], { hour: 'numeric' }).resolvedOptions().hour12
+  const getTimeFormat = (hour: number): string => {
```

Comment:
You need t opass ampm as parameter to getTimeFormat and should we do internationalisation?